v2.0/paths/task_assignments__put_task_assignments_id.yml

```diff
@@ -46,3 +46,10 @@ responses:
       application/json:
         schema:
           $ref: '#/components/schemas/TaskAssignment'
+  400:
```

Comment:
Does this really return a TaskAssignment object on 404?

Can we make the description match the format “Returns an error if…” as per https://github.com/box/box-openapi/blob/master/v2.0/paths/chunked_uploads__post_files_id_upload_sessions.yml#L51?